lib/src/compile.test.ts

```diff
@@ -61,6 +61,36 @@ describe('compile', () => {
         expect(await compile({path})).toBe('a {\n  b: c;\n}');
         await fs.unlink(path);
       });
+
+      it('compiles imports using includePaths', async () => {
```

Comment:
Please also add a spec in `render.test.ts`.

Reply:
Added. The render tests use a sandbox wrapper that the compile tests don't. Should I be using it in compile.test.ts? It seems to handle the folder/file cleanup.